src/cli.rs

```diff
@@ -33,6 +35,7 @@ use crate::{buffer::Buffer, request_items::RequestItem};
         AppSettings::DeriveDisplayOrder,
         AppSettings::UnifiedHelpMessage,
         AppSettings::ColoredHelp,
+        AppSettings::AllArgsOverrideSelf,
```

Comment:
Good catch, I didn't realize we didn't already have this behavior.

Reply:
I also didn't know a flag like this existed until I read https://github.com/clap-rs/clap/discussions/2627